site/docs/v1/tech/developer-guide/api-gateways/aws-api-gateway.adoc

```diff
@@ -0,0 +1,217 @@
+
+Under [field]#Security#, choose `Create a JWT authorizer`.
+
+Set [field]#Identity source# to `$request.header.Authorization`.
+
+For [field]#Issuer#, set the URL to be the same as you set in FusionAuth for the JWT issuer in the Tenant configuration. Make sure that it is exactly the same, including trailing slashes, without any whitespace.
+
+Under [field]#Audiences#, paste in the `Client ID` from the FusionAuth application created earlier.
+
+Then click the primary `Add` button at the bottom right of the screen.
+
+After completing this, a new route will be added to your gateway API, with the same name as the lambda, `privateFunction` here. Concat this name with the "Invoke URL" recorded earlier when setting up the API gateway to get the full URL. It should look similar in form to this: `\https://w3miabt10d.execute-api.us-east-2.amazonaws.com/privateFunction`.
+
+If you call this route through a browser, Postman, or cURL, you should receive a 401 and an `Unauthorized` message. We'll create a token next with FusionAuth to show how the route can be successfully called.
+
+== Testing with a token from FusionAuth
+
+To access the secured lambda function, we'll need to get a valid token from FusionAuth. Normally, your frontend app would redirect a user to FusionAuth to login, obtain a code, and then exchange that code for a token, using a framework of your choice. For the purposes of this guide, we'll call the FusionAuth API on behalf of a user to create a token directly.
+
+To enable this, there are a few settings that need to be changed on FusionAuth. In the side panel in FusionAuth, select [breadcrumb]#Settings > API Keys#. Click the green `+` button at the top left to create a new key.
+
+Give the key a meaningful description, like `API Gateway Key`. Select the Tenant that you created the FusionAuth application under (normally `Default`).
+
+Then, under [breadcrumb]#Endpoints#, allow `POST` on the `/api/login` route.
+
+image::api-gateways/aws-api-gateway/login-api.png[Enabling the login API,width=1200]
+
+Save the API key. You should now see a list of all API keys. Click on the lock icon next to the [field]#key# field for the key you just created and copy out the API key for later use.
+
+image::api-gateways/aws-api-gateway/api-key.png[Enabling the login API,width=1200,role=bottom-cropped]
+
+Now create a test user to login with on FusionAuth. Navigate to [breadcrumb]#Users# in the sidebar and click the green `+` button at the top right of the screen to create a new user. Choose the appropriate tenant (usually `default`), and fill in the information. You can choose not to send an email to set up the password but rather set it directly when creating the user by toggling the [field]#Send email to set up password# switch. Record this user email and password, as we'll use it to obtain a token.
```

Comment:
Any reason to set up a test user? Any reason not to use the current user (since they were able to log into FusionAuth?)